### ISSUE TITLE
HealPlates - Extend Threat Plates to behave similarly for low health friendlies

### ISSUE THREAD

**Issue by jonathanhupp** [Backupiseasy/ThreatPlates#255]
tl;dr; As a healer, I want friendly plates to get big and red when their health gets low, in the same way they do when threat gets high.

I'd like to be able to separately scale, color, and decorate friendly units based on their health without those rules applying to enemy plates. Currently I can color _all_ plates by health, but I only want friendlies to change based on health. Color based on reaction for enemies should be able to work in conjunction.

I'm not sure how difficult this would be, but I figure all of the tools are already there, just need the options.

I realize most people probably just use VuhDo or healbot, which is fine and I do too, but I want to try something new. All of the functionality for this already exists for threat, it just would need to be based on health instead (and have some way to also account for threat, as unlikely as that may be).
